src/block-ciphers.rst

```diff
@@ -220,51 +220,51 @@ A closer look at Rijndael
 .. canned_admonition::
    :from_template: advanced
 
-AES consists of several independent steps. At a high level, AES is a
+AES includes several independent steps. At a high level, AES is a
 :term:`substitution-permutation network`.
 
 Key schedule
 ''''''''''''
 
-AES requires separate keys for each round in the next steps. The key
-schedule is the process which AES uses to derive 128-bit keys for each
+The next steps show how AES requires separate keys for each round. In the key
+schedule process, AES derives 128-bit keys for each
 round from one master key.
 
-First, the key is separated into 4 byte columns. The key is rotated and
-then each byte is run through an S-box (substitution box) that maps it
+First, the key separates into 4 byte columns. The key rotates and
+each byte runs through an S-box (substitution box), which maps it
 to something else. Each column is then XORed with a round constant. The
 last step is to XOR the result with the previous round key.
 
-The other columns are then XORed with the previous round key to produce
+The next columns are XORed with the previous round key to produce
 the remaining columns.
 
 SubBytes
 ''''''''
 
-SubBytes is the step that applies the S-box (substitution box) in AES.
-The S-box itself substitutes a byte with another byte, and this S-box is
+The AES SubBytes step applies to the S-box (substitution box).
```

Comment:
This changes the meaning of the sentence to be incorrect. SubBytes _applies the S-box_. It does not apply _to_ the S-box. SubBytes is itself an application of the S-box.